can/interfaces/__init__.py

```diff
@@ -50,7 +47,8 @@
     else:
         BACKENDS.update(
             {
-                interface.name: tuple(interface.value.split(":"))
+                # This cast in wrong if interface.value is formatted badly, but we just fail later
+                interface.name: cast(Tuple[str, str], tuple(interface.value.split(":")))
```

Comment:
```suggestion
                interface.name: cast(Tuple[str, str], tuple(interface.value.split(":", maxsplit=1)))
```

Reply:
Yup, thanks!

Reply:
Alternatively we could use some helper function so it actually fails when there are more or less than two values:

```python3
        def _read_entry_point(value: str) -> Tuple[str, str]:
            """Helper function to satisfy mypy."""
            _module, _class = value.split(":")
            return _module, _class
```
But it's very verbose